### DISCUSSION TITLE
Is it possible to set the terminal background using oh-my-posh?

### DISCUSSION THREAD

**Discussion by Lockszmith-GH** [JanDeDobbeleer/oh-my-posh#4100] [Troubleshoot] [answered]
Since omp sets the terminal's colors using ANSI Codes, I thought it should be 'easy' to have omp dictate defautl background color for a terminal, but I can't seem to find such a solution.

Any ideas of how to achieve this? (for example setting the background to red)

**Comment by JanDeDobbeleer** [comment, marked as answer]
@Lockszmith-GH that's not possible. There's no API to control the terminal background other than the terminal's settings.

**Reply by Lockszmith-GH** [reply]
Damn, I was hoping that there was a way to "not reset" the background.
While I undsrstand that's not perfect.

Anyway, thanks for an awesome tool.

**Reply by Daksol** [reply]
Indeed a pity that not possible. I was keen to have Terminal show a different background when Powershell is launched in Administration mode. While out of box Windows gives you the shield icon top left, and OhMyPosh gives you the "flash!" icon in the prompt, I would like to have a more visually impactful reminder that commands in such a window need even more care!

**Reply by JanDeDobbeleer** [reply]
@Daksol You could ask the Windows Terminal team for this feature 😃
